### PR TITLE
Fix Sign Up Log In Bugs

### PR DESCRIPTION
**Pull request by thejoefriel**
#### Issues this PR relates to
#413 #437 #434  #439 #433

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/Route/index.js [hunk 1]
```diff
@@ -48,6 +55,26 @@ const Route = (props) => {
     }
     return <Redirect to={GENERAL.LOG_IN} />;
   }
+
+  if (isLoggedIn && path === GENERAL.LOG_IN) {
+    console.log("reached");
```
Comment (RamyAlshurafa) [inline client/src/components/Route/index.js:60]:
```suggestion
    console.log("reached");
```

File: client/src/pages/AcceptTimeCredits/index.js [hunk 2]
```diff
@@ -42,6 +42,11 @@ const AcceptTimeCredits = (props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const handleID = (value) => {
+    const cleanID = value.trim();
```
Comment (RamyAlshurafa) [inline client/src/pages/AcceptTimeCredits/index.js:46]: this need to lowercase the membership id

File: client/src/components/Route/index.js (hunk 1 above)
Comment (thejoefriel) [reply client/src/components/Route/index.js:60]: done

File: client/src/pages/AcceptTimeCredits/index.js (hunk 2 above)
Comment (thejoefriel) [reply client/src/pages/AcceptTimeCredits/index.js]: done

**Pull request merged**